website_logo/readme/USAGE.rst

```diff
@@ -0,0 +1,4 @@
+To use this module, you need to:
+
```

Comment:
Remove this two lines.

Reply:
> And I was guided by this.
> https://github.com/OCA/maintainer-tools/blob/master/template/module/readme/CONFIGURE.rst
> Should I remove those two lines anyway?


and here https://github.com/OCA/maintainer-tools/blob/master/template/module/readme/USAGE.rst